javascript/ql/src/Expressions/UselessDefensiveProgramming.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @name Useless defensive code
+ * @description If the situation some defensive code guards against never
```

Comment:
Sorry but I don't understand what you mean in the description. Would it be possible to rephrase so it's clearer?